clients/extension/src/content/index.ts

```
@@ -1,1303 +1,20 @@
-import { announceProvider, EIP1193Provider } from 'mipd'
-import { v4 as uuidv4 } from 'uuid'
+import { setupBridgeMessengerRelay } from '../messengers/bridge'
 
-enum NetworkKey {
-  MAINNET = 'mainnet',
-  TESTNET = 'testnet',
+// This fake import is for Knip static analysis only.
+// It ensures that dependencies inside inpage.ts are not marked as unused.
+// It has no effect on runtime because the script is injected dynamically.
+// eslint-disable-next-line no-constant-condition
+if (false) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)